### PR TITLE
feat(condo): INFRA-210 apollo query batching

### PR DESCRIPTION
**Pull request by sitozzz**
- [x] remove outdated package `express-graphql` and use `apollo-server-express` instead
- [x] move stitchSchema to packages and reuse in all involved miniapps
- [x] update submodules to master when ready

### CODE REVIEW

**Review by toplenboren — APPROVED** [review]

File: packages/keystone/stitchSchema.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            uploads: false,
+        })
+
+        app.use(
+            this.apiUrl,
+            bodyParser.json(),
+            graphqlUploadExpress({ maxFiles: MAX_FILES, maxFileSize: MAX_FILE_SIZE }),
+        )
+
+        app.use(server.getMiddleware({ path: this.apiUrl, cors: false }))
```
Comment (SavelevMatthew) [inline packages/keystone/stitchSchema.js:213]: What's the `cors: false` stands for?

**Review by SavelevMatthew — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]

File: packages/keystone/stitchSchema.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                return { ...await fetchResult.json(), status, statusText }
+            } catch (error) {
+                return { status, statusText }
+            }
+        }
+    }
+}
+
+async function makeGatewaySchema (appTokenKey, condoAccessTokenKey) {
+    const appAPIUrl = `${conf.PASS_DOMAIN}/admin/api`
```
Comment (Alllex202) [inline packages/keystone/stitchSchema.js:141]: `PASS_DOMAIN`? :)

[the remark this answers is not in the document]
Comment (sitozzz) [reply packages/keystone/stitchSchema.js:141]: Oops, thnks for pointing that

File: packages/keystone/stitchSchema.js [hunk 3]
```diff
@@ -0,0 +1,220 @@
+const { GraphQLFileLoader } = require('@graphql-tools/graphql-file-loader')
```
Comment (pahaz) [inline packages/keystone/stitchSchema.js:1]: need to add it to package.json

**Pull request merged**